src/foam/u2/filter/DateFilterView.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.filter',
+  name: 'DateFilterView',
+  extends: 'foam.u2.Controller',
+
+  documentation: `A SearchView for properties of type Date.`,
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  css: `
+    ^ {
+      padding: 24px 16px;
+      box-sizing: border-box;
+      min-width: 214px;
+    }
+
+    ^ .foam-u2-tag-Select {
+      width: 100%;
+      height: 36px;
+
+      border-radius: 3px;
+      border: solid 1px #cbcfd4;
+      background-color: #ffffff;
+    }
+
+    ^ .foam-u2-DateView {
+      width: 100%;
+      height: 36px;
+
+      margin-top: 16px;
+
+      border-radius: 3px;
+      border: solid 1px #cbcfd4;
+      background-color: #ffffff;
+    }
+  `,
+
+  properties: [
+    {
+      name: 'property',
+      documentation: `The property that this view is filtering by. Should be of
+          type Date.`,
```

Comment:
You say - 'should be of type date' .. do you want to add validation to enforce the 'should'?

Reply:
The way it is currently implemented, the developer shouldn't be setting the property. The act of defining a `date` property in their models should be enough.